scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if command -v az &> /dev/null
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "Azure CLI is already installed at $(command -v az)"
+    fi
+  else
+    doRequireNotBatch
+    if doIsWindows
+    then
+      # Get leatest release
+      if [ -z "${AZURECLI_VERSION}" ]
+      then
+        doEcho "Getting latest release..."
+        AZURECLI_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
+      fi
+      local version="${AZURECLI_VERSION}"
+      doDownload "-" "" "az" "${version}"
+      # Install Azure CLI for Windows
+      doEcho "Installing Azure CLI for Windows..."
+      installOnWindows
+    else
+      doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+    fi
+  fi
+}
+
+# Install Azure CLI for Windows via Powershell Command
+function installOnWindows() { 
+  winPath=$(cygpath -w "${DEVON_DOWNLOAD_DIR}")
+  powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${winPath}\az-${version}-windows.msi /QB-!'" &> /dev/null
+  if [ "${?}" = 0 ]
+  then 
+    setConfigDir
+    doEcho "Azure CLI has been installed and is ready to use. \n\n******** IMPORTANT ******** \nRestart the console to use Azure CLI regular." 
+  else
+    if doAskToContinue "\nAdministration rights are required. Do you want to restart the installation?"
+    then
+      doEcho "\nInstalling Azure CLI for Windows..."
+      installOnWindows
+    else
+      doEcho "The installation has been canceled."
+    fi
+  fi
+}
+    
+# Set directory for azure cli configuration file
+function setConfigDir() {
+  azureconf_dir="${DEVON_IDE_HOME}/conf/.azure"
+  azurecli_export="export AZURE_CONFIG_DIR=${azureconf_dir}"
+  if ! grep -q "${azurecli_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      doRunCommand "${azurecli_export}"
+      echo -e "\n${azurecli_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
```

Comment:
very nice with leading newline to prevent errors if the file is not terminated with newline. :)